src/Result/StoreEmailSettings.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+class StoreEmailSettings extends AbstractResult
+{
+    public function getServer(): string
+    {
+        $data = $this->getData();
+        return $data['server'];
+    }
+
+    public function getPort(): string
+    {
+        $data = $this->getData();
+        return $data['port'];
+    }
+
+    public function getUsername(): string
+    {
+        $data = $this->getData();
+        return $data['login'];
+    }
+
+    public function getPassword(): string
+    {
+        $data = $this->getData();
+        return $data['password'];
+    }
+
+    public function getFromEmail(): string
+    {
+        $data = $this->getData();
+        return $data['from'];
+    }
+
+    public function getFromName(): string
```

Comment:
method name doesn't match field name, not consistent? even though your method name is better than the field, might not be able to find the method in the IDE if you expect the field name

Reply:
yes, that's why we explicitly state it is an opinionated library, like this example the naming on the api is a bit confusing for mail/smtp connections: 'login', 'from', 'fromDisplay'; The idea was to not make the users need to check the API docs to find out what the variables mean. Not sure if mine are less confusing than the original ones, lmk if you have better suggestions.

Reply:
No your name is better, I would name it the same way. Let's stick with it.